src/sqlancer/cnosdb/CnosDBComparatorHelper.java

```diff
@@ -0,0 +1,163 @@
+package sqlancer.cnosdb;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+
+import sqlancer.IgnoreMeException;
+import sqlancer.cnosdb.client.CnosDBResultSet;
+import sqlancer.cnosdb.query.CnosDBSelectQuery;
+import sqlancer.common.query.ExpectedErrors;
+
+public final class CnosDBComparatorHelper {
+
+    private CnosDBComparatorHelper() {
+    }
+
+    public static boolean isEqualDouble(String first, String second) {
```

Comment:
For this and other methods, is there actually a need to re-implement them? It seems that, at least for this method and the one below, the code is the same as for https://github.com/sqlancer/sqlancer/blob/67751dc3d83b880cd8c7ed53416b8b157a2ba2ea/src/sqlancer/ComparatorHelper.java#L21-L37